src/app/components/apps-view/apps-view.component.html

```diff
@@ -85,10 +71,16 @@
       <ng-container [matColumnDef]="columnDef.colId" *ngFor="let columnDef of appColumnDef">
         <mat-header-cell *matHeaderCellDef mat-sort-header>{{ columnDef.colName }}</mat-header-cell>
 
-        <ng-container *ngIf="columnDef.colId === 'submissionTime'; else renderNext_1">
+        <ng-container *ngIf="columnDef.colId === 'submissionTime'; else renderNext_4">
```

Comment:
I think we can do better rendering for submissionTime and lastStateChangeTime.
1. Move formatTimeIfNotNull function to common-util file as timeColumnFormatter. Use this as colFormatter for submissionTime as well as lastStateChangeTime.
2. Here check for colFormatter and use innerText to render the text. Like:
```
        <ng-container *ngIf="columnDef.colId === 'usedResource'; else renderNext_1">
          <mat-cell *matCellDef="let element">
            <ng-container *ngIf="columnDef.colFormatter; else showAppRawData">
              <span [innerHTML]="columnDef.colFormatter(element[columnDef.colId])"></span>
            </ng-container>
            <ng-template #showAppRawData>
              <span>{{ element[columnDef.colId] }}</span>
            </ng-template>
          </mat-cell>
        </ng-container>

        <ng-template #renderNext_1>
          <ng-container *ngIf="columnDef.colFormatter; else renderNext_2">
          <mat-cell *matCellDef="let element">
            <span [innerText]="columnDef.colFormatter(element[columnDef.colId])"></span>
          </mat-cell>
        </ng-container>
        </ng-template>

        <ng-template #renderNext_2>
          <mat-cell *matCellDef="let element">{{ element[columnDef.colId] || 'n/a' }}</mat-cell>
        </ng-template>
      </ng-container>
```

Reply:
Thanks for the suggestion, updated!